File: user_spring/src/main/java/com/kjj/user/dto/order/LastOrderDto.java
package com.kjj.user.dto.order;

import com.kjj.user.entity.Order;
import com.kjj.user.util.DateUtil;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class LastOrderDto {
    private Long id;

    private String menu;
    private int cost;
    private String orderDate;
    private boolean recognize;

    public static LastOrderDto from(Order order) {
        return new LastOrderDto(
                order.getId(),
                order.getMenu(),
                order.getCost(),
                DateUtil.makeLocaldateToFormatterString(order.getOrderDate()),
                order.isRecognize()
        );
    }
}

File: user/src/main/kotlin/com/kjj/user/dto/order/LastOrderDto.kt
package com.kjj.user.dto.order

import com.kjj.user.entity.Order
import com.kjj.user.util.DateUtil

data class LastOrderDto(
    val id: Long?,
    val menu: String,
    val cost: Int,
    val orderDate: String,
    val recognize: Boolean
) {
    companion object {
        fun from(order: Order): LastOrderDto {
            return LastOrderDto(
                order.id,
                order.menu,
                order.cost,
                DateUtil.makeLocalDateToFormatterString(order.orderDate),
                order.recognize
            )
        }
    }
}